certification/formatters/generic_test.go

```diff
@@ -0,0 +1,168 @@
+package formatters
```

Comment:
Any particular reason to not use ginkgo/gomega? The rest of the code base is being tested with that framework.

Reply:
I can re-write in Ginkgo if desired.  Didn't know it was a requirement.

Reply:
Not a hard requirement. Just curious, really.

Reply:
I just don't prefer the Ginkgo syntax but I'm good with it either way.

Reply:
Yep, I think we probably prefer to keep things consistent, but this doesn't introduce any other frameworks other than gotest, and it's a relatively small package. I could go either way. I'm fine accepting it as is.

Reply:
Thanks!